sandbox/RazorLight.Sandbox/RazorLight.Sandbox.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+      <TargetFrameworks>net5.0</TargetFrameworks>
```

Comment:
This is fine.  It makes sense to me the sandbox should primarily be for experimenting with the latest version as that is where most new issues will come in.